src/applications/disability-benefits/all-claims/tests/content/fileUploadDescription.unit.spec.jsx

```diff
@@ -7,27 +7,15 @@ import { MAX_FILE_SIZE_MB, MAX_PDF_FILE_SIZE_MB } from '../../constants';
 
 describe('526 All Claims -- File upload descriptions', () => {
   it('should render', () => {
-    const tree = shallow(
-      <UploadDescription uploadTitle={'test'} showPdfSize={false} />,
-    );
+    const tree = shallow(<UploadDescription uploadTitle="test" />);
     expect(tree.find('UploadDescription')).to.exist;
     tree.unmount();
   });
-  it('should render only the max global file size', () => {
-    const tree = shallow(
-      <UploadDescription uploadTitle={'test'} showPdfSize={false} />,
-    );
+  it('should render the max file sizes', () => {
+    const tree = shallow(<UploadDescription uploadTitle="test" />);
     const text = tree.text();
-    expect(text).to.contain(`Maximum file size: ${MAX_FILE_SIZE_MB}`);
-    expect(text).to.not.contain(MAX_PDF_FILE_SIZE_MB);
-    tree.unmount();
-  });
-  it('should render only the appropriate panels', () => {
```

Comment:
Removed this test since it had a similar assertion to the one above after rewriting for removing the feature flag.